cmd/drainer/drainer.toml

```diff
@@ -120,6 +120,11 @@ port = 3306
 # ssl-key = "/path/to/drainer-key.pem"
 # The common name which is allowed to connection with cluster components.
 # cert-allowed-cn = ["binlog"]
+#
+# rabbitmq syncer config
```

Comment:
could you link some doc here (noticed that you use `github.com/streadway/amqp`) or give some explanation about these settings?